src/todoApp.js

```diff
@@ -0,0 +1,42 @@
+const todoApp = { 
+
+    // if for keeping track of each object valule
+    id : 0,
+
+    todos: [],
+
+    create : function (text) { 
+
+        this.id++ // incrementing the value 
+
+        const todo = {
+            id: this.id,
+            text: text,
+            status : "incomplete"
+        }
+
+        this.todos.push(todo)
+        return todo
+    }, 
+
+    showAll: function() { 
+        return this.todos
+    },
+
+    setCompleted: function(id){
+
+        // const incomplete = todos.map((todo) => todo.status === "incomplete")
+
+        const todo = this.todos.find(todo => todo.id )
```

Comment:
This find() method could be improved. What you would get if you'd check the console.log for todo here?